lib/lib-table-query/src/main/java/org/sagebionetworks/table/query/model/CurrentUserFunction.java

```diff
@@ -0,0 +1,25 @@
+package org.sagebionetworks.table.query.model;
+
+import java.util.List;
+
+public class CurrentUserFunction extends SQLElement{
+
+    CurrentUserFunctionName currentUserFunctionName;
+
+    public CurrentUserFunction(CurrentUserFunctionName currentUserFunctionName){
+        this.currentUserFunctionName = currentUserFunctionName;
+    }
+
+    @Override
+    public void toSql(StringBuilder builder, ToSqlParameters parameters) {
+        builder.append(currentUserFunctionName.name());
```

Comment:
This can be hard-coded to CURRENT_USER(), unless you need it somewhere else. Might it be that this is a HasFunctionReturnType? (e.g. like the MySqlFunction). How is the return type used? Is it used if the function is in the select clause? E.g. "select current_user() from ..." would need some sort of way to identify the type returned by the select.